lang/scala/scala.talon

```diff
@@ -0,0 +1,37 @@
+mode: user.scala
+mode: user.auto_lang
+and code.language: scala
+-
+tag(): user.code_operators
+tag(): user.code_comment
+tag(): user.code_block_comment
+tag(): user.code_generic
+
+settings():
+    user.code_private_function_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_protected_function_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_public_function_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_private_variable_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_protected_variable_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_public_variable_formatter = "PRIVATE_CAMEL_CASE"
+
+state package: "package "
+
+state val: "val "
+state var: "var "
+
+state match: user.code_state_switch()
+
+state case class: "case class "
+state object: "object "
+
+string: key('"')
+char: key("'")
+block string:
```

Comment:
Multiline strings with a special syntax are also in Perl, bash, Python, Ruby etc. If we end up making this more generic to encompass them at some point it'd be good to be able to preserve the command name.

I think this name is fine, but want to draw this to other maintainer's attention.